play/plugin/src/test/kotlin/com/github/triplet/gradle/play/tasks/GenerateEditIntegrationTest.kt

```diff
@@ -72,6 +72,8 @@ class GenerateEditIntegrationTest : IntegrationTestBase(), SharedIntegrationTest
                             .named("playApi-com.example.publisher")
                             .get().service.get() as PlayApiService
 
+                service.cleanupAtClose = false
```

Comment:
Added `cleanupAtClose` just for this test, as it's checking the `editFile` after the build is gone. As now the service closes before the build ends, this file is getting deleted.

Reply:
Actually I just needed to add `usesService(service)` here and not `service.cleanupAtClose = false`